tests/e2e/framework/kubernetes.go

```diff
@@ -144,42 +144,57 @@ func (k *KubeInfo) Teardown() error {
 	glog.Info("Cleaning up kubeInfo")
 	var err error
 
-	if *rbacfile != "" {
+	if *skipSetup {
+		return err
```

Comment:
return nil in that case, even though err is nil, just make it explicit.

Reply:
ok